example/time/time_global_test.c

```diff
@@ -4,6 +4,14 @@
  * SPDX-License-Identifier:     BSD-3-Clause
  */
 
+/**
+ * @example time_global_test.c
```

Comment:
Is this visible if you generate it locally?

Reply:
Good catch, I was wondering the same thing at some point but completely forgot it. Perhaps the cleanest fix is to rename the application to e.g. odp_time_global_test, so it matches Doxyfile FILE_PATTERNS.

Reply:
Renamed to `odp_time_global_test.c ` in V2.